web-console/src/views/tasks-view.tsx

```diff
@@ -293,9 +298,38 @@ ORDER BY "rank" DESC, "created_time" DESC`);
     </AsyncActionDialog>;
   }
 
+  private initTableColumnSelection() {
+    if (localStorage.getItem(supervisorTableColumnSelection) == null) {
+      const columns: string[] = ["Datasource", "Type", "Topic/Stream", "Status", "Actions"];
+      const defaultSetting: any = {};
+      columns.forEach(column => defaultSetting[column] = true);
+      localStorage.setItem(supervisorTableColumnSelection, JSON.stringify(defaultSetting));
+    }
```

Comment:
I see these 6 lines of code repeated a lot... really feels like you could make a sweet utility function here.